common/utils/works.js

```diff
@@ -11,6 +15,15 @@ export function getPhysicalLocations(work: Work) {
     .reduce((acc, locations) => acc.concat(locations), []);
 }
 
+export function getRepositoryFromIIIFPresentationManifest(
+  iiifManifest: IIIFManifest
+): ?IIIFMetadata {
```

Comment:
To make this more generic, and keep naming and type together, we could have a `getIIIFMetadata(label: string): IIIFMetaData`?